digdag-standards/src/main/java/io/digdag/standards/command/DockerCommandExecutor.java

```diff
@@ -70,6 +70,9 @@ private Process startWithDocker(Path projectPath, TaskRequest request, ProcessBu
         }
         else {
             imageName = baseImageName;
+            if (dockerConfig.has("pull-always")) {
```

Comment:
It seems this option should work as boolean so that users can set `false` to disable this feature. How about this one?
```
            if (dockerConfig.get("pull-always", Boolean.class, false)) {
                pullImage(imageName);
            }
```